plugins/polio/js/src/domains/Campaigns/hooks/api/useGetCampaignTypes.ts

```diff
@@ -20,7 +20,7 @@ export const useGetCampaignTypes = (): UseQueryResult<
                 return (
                     data?.map((campaignType: CampaignType) => {
                         return {
-                            value: campaignType.id,
+                            value: campaignType.slug,
```

Comment:
useGetCampaignTypes is also used in BaseInfoForm, so by removing the compaignType.id it crush the OBR Name renaming when the campaignType isPolio == false
![Screenshot from 2024-03-26 14-45-47](https://github.com/BLSQ/iaso/assets/19631540/adbe23ed-a14a-4bd3-8bc0-9a26d1bae95f)